runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -520,6 +522,60 @@ J9::ARM64::TreeEvaluator::arraylengthEvaluator(TR::Node *node, TR::CodeGenerator
    return lengthReg;
    }
 
+TR::Register *
+J9::ARM64::TreeEvaluator::ZEROCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   // NOTE: ZEROCHK is intended to be general and straightforward.  If you're
+   // thinking of adding special code for specific situations in here, consider
+   // whether you want to add your own CHK opcode instead.  If you feel the
+   // need for special handling here, you may also want special handling in the
+   // optimizer, in which case a separate opcode may be more suitable.
+   //
+   // On the other hand, if the improvements you're adding could benefit other
+   // users of ZEROCHK, please go ahead and add them!
+   //
+   // If in doubt, discuss your design with your team lead.
```

Comment:
I'm not sure this entire comment section is warranted.  What it is preaching certainly doesn't apply to just the ZEROCHK opcode.  Please remove the whole thing.

Reply:
Remove the comment.
`ZEROCHKEvaluator()` for other platforms, such as x and p, also have the same comment.